src/common/_modules/app_continua/autoscaler.tf

```
@@ -0,0 +1,62 @@
+resource "azurerm_monitor_autoscale_setting" "appservice_continua" {
+  name                = format("%s-autoscale", module.appservice_continua_itn.app_service.app_service.name)
+  resource_group_name = azurerm_resource_group.continua_itn_rg.name
+  location            = var.location_itn
+  target_resource_id  = module.appservice_continua_itn.app_service.plan.id
+
+  profile {
+    name = "default"
+
+    capacity {
+      default = 1
+      minimum = 1
+      maximum = 10
+    }
+
+    rule {
+      metric_trigger {
+        metric_name              = "CpuPercentage"
+        metric_resource_id       = module.appservice_continua_itn.app_service.plan.id
+        metric_namespace         = "microsoft.web/serverfarms"
+        time_grain               = "PT1M"
+        statistic                = "Average"
+        time_window              = "PT5M"
+        time_aggregation         = "Average"
+        operator                 = "GreaterThan"
+        threshold                = 70
+        divide_by_instance_count = false
+      }
+
+      scale_action {
+        direction = "Increase"
+        type      = "ChangeCount"
+        value     = "5"
+        cooldown  = "PT5M"
+      }
+    }
+
+    rule {
+      metric_trigger {
+        metric_name              = "CpuPercentage"
+        metric_resource_id       = module.appservice_continua_itn.app_service.plan.id
+        metric_namespace         = "microsoft.web/serverfarms"
+        time_grain               = "PT1M"
+        statistic                = "Average"
+        time_window              = "PT20M"
+        time_aggregation         = "Average"
+        operator                 = "LessThan"
+        threshold                = 30
+        divide_by_instance_count = false
+      }
+
+      scale_action {
+        direction = "Decrease"
+        type      = "ChangeCount"
+        value     = "5"
```

Scale down by `5` is dangerous, usually this value should be `1`
```suggestion
        value     = "1"
```

---

[727732d](https://github.com/pagopa/io-infra/pull/1661/commits/727732d490caa2acf525c029bc7db247e9631b96)